csharp/Managed/Messaging.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using LibPDBinding.Native;
+using LibPDBinding.Managed.Events;
+using LibPDBinding.Managed.Data;
+using LibPDBinding.Managed.Utils;
+using System.Runtime.CompilerServices;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LibPDBinding
+{
+	/// <summary>
+	/// Messaging in Pd.
+	/// </summary>
+	public sealed class Messaging
```

Comment:
To minimize methods on a class, I have created a class simple for messaging, that is created with the Pd instance and cannot be created from a user (internal constructor)